src/style.css

```diff
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+}
+
+.input_mockup_style {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big_form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big_icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  right: 17px;
+  width: 19px;
+  height: 19px;
+  z-index: 1;
+}
+
+.big_input {
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  outline: none;
+}
+
+.small_form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.small_icon {
+  position: absolute;
+  top: 15px;
+  right: 9px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+  z-index: 1;
+}
+
+.small_input {
+  padding-left: 33px;
+  padding-bottom: 3px;
+  width: 100%;
+  height: 42px;
+  outline: none;
+}
+
+.big_form:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.big_form:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.big_input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.small_form:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
```

Comment:
Avoid code duplication, if you need to give the same styles to two elements when hovering - use the following construction:

```
.classA:hover, .classB:hover {
    // styles
}
```